site/docs/pages/transaction/transaction.mdx

```diff
@@ -310,6 +310,7 @@ Obtain a Paymaster and Bundler endpoint from the [Coinbase Developer Platform](h
 import { Transaction, TransactionButton, TransactionSponsor } from "@coinbase/onchainkit/transaction"
 // ---cut-before---
 // ommited for brevity
+// @errors: 2322
```

Comment:
think you'll need @noErrors: 2322 to completely suppress the error

Reply:
Just did